File: ASSIGNEMENT/MODULE 03/STRING/Q14.cpp
// Q(14) Write a program in C to combine two strings manually....

#include<stdio.h>

main()
{
	char a[30],b[30];
	int i,j;
	
	printf("enter A string:");      				// inputs.....A...
	gets(a);
	
	printf("\nenter B string:");					// input......B...
	gets(b);
	
	for(i=0;a[i]!='\0';i++);					// using loop for A...
	
	for(j=0;b[j]!='\0';j++)						// using loop for B....
	{
		a[i]=b[j];								// stored in a.
		i++;
	}
	a[i]='\0';
	
	printf("\ncombine two strings: %s ",a);
	
}

File: PYTHON PROGRAMMING/ASSIGNMENTS/Module – 2 (Fundamentals of python)/Q14.py
#Write a Python program to get a single string from two given strings, 
                #separated by a space and swap the first two characters of each string...

n=input("Enter string1: ")
n2=input("Enter string2: ")

                    # give conditon to greater than 2..
if len(n)<2 or len(n2)<2:

    print("both string atleast two character..")

else:
                #swapping the values...
    new_n= n[:2]+ n2[2:]
    new_n2= n2[:2]+ n[2:]

print(new_n," ",new_n2)
